sdcm/nemesis.py

```diff
@@ -176,6 +176,7 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     kubernetes: bool = False        # flag that signal that nemesis run with k8s cluster
     limited: bool = False           # flag that signal that nemesis are belong to limited set of nemesises
     has_steady_run: bool = False    # flag that signal that nemesis should be run with perf tests with steady run
+    cloud_sanity: bool = False      # flag signaling that nemesis should be run as part of siren-tests sanity longevity
```

Comment:
why no set those from the siren-test code ? 
i.e. this is a filter that is used by the siren-test code, isn't more logical that the flags would be set by siren-test, and not in SCT ?

Reply:
maybe just something like that in siren-test sct-plugin ?
```pyhton
from sdcm.nemesis import StopStartMonkey

StopStartMonkey.cloud_sanity = True
```

it's just need to be a property ontop of the class, the nemesis code doesn't really care how it got there.

I'm just saying that the labels should be defined in one place, not scattered between both. i.e. if I want to understand what part of `cloud_sanity` I should be looking at one code base, and not multiple of them.

Reply:
I moved all the things related to the nemesis filter to `siren-tests`. Here I left only the commit for the exception printing fix.
Renamed the PR